templates/pages/admin/entity/custom_ui.html.twig

```diff
@@ -0,0 +1,85 @@
+{#
+ # ---------------------------------------------------------------------
+ #
+ # GLPI - Gestionnaire Libre de Parc Informatique
+ #
+ # http://glpi-project.org
+ #
+ # @copyright 2015-2023 Teclib' and contributors.
+ # @copyright 2003-2014 by the INDEPNET Development Team.
+ # @licence   https://www.gnu.org/licenses/gpl-3.0.html
+ #
+ # ---------------------------------------------------------------------
+ #
+ # LICENSE
+ #
+ # This file is part of GLPI.
+ #
+ # This program is free software: you can redistribute it and/or modify
+ # it under the terms of the GNU General Public License as published by
+ # the Free Software Foundation, either version 3 of the License, or
+ # (at your option) any later version.
+ #
+ # This program is distributed in the hope that it will be useful,
+ # but WITHOUT ANY WARRANTY; without even the implied warranty of
+ # MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ # GNU General Public License for more details.
+ #
+ # You should have received a copy of the GNU General Public License
+ # along with this program.  If not, see <https://www.gnu.org/licenses/>.
+ #
+ # ---------------------------------------------------------------------
+ #}
+
+{% extends "generic_show_form.html.twig" %}
+{% import 'components/form/fields_macros.html.twig' as fields %}
+{% set params  = params ?? [] %}
+
+{% block form_fields %}
+    {% block more_fields %}
+        {{ fields.dropdownArrayField('enable_custom_css', item.fields['enable_custom_css'], enable_css_options, __('Enable CSS customization')) }}
+        <div id="custom_css_container" class="custom_css_container {{ item.fields['enable_custom_css'] == 1 ? '' : 'd-none' }}" style="height: 400px"></div>
+        <div class="alert alert-info {{ item.fields['enable_custom_css'] == 1 ? 'd-none' : '' }}">{{ __('Custom CSS is disabled') }}</div>
```

Comment:
On a sub-entity, this message is displayed when the "Inherits configuration from the parent entity" is selected. This is not corresponding to the effective configuration when the parent entity has custom CSS enabled.
Is it possible to display the inherited value?